packages/design/src/colors.css

```diff
@@ -167,8 +165,6 @@
   --color-indigo--lightest: rgb(230, 233, 247);
   --color-indigo--dark: rgb(55, 69, 132);
 
-  /* TODO: Reintroduce RGB once the issue with JobberStyle is fixed */
-  /* --color-white--rgb: 255, 255, 255; */
-  /* --color-white: rgba(var(--color-white--rgb)); */
-  --color-white: rgb(255, 255, 255);
+  --color-white--rgb: 255, 255, 255;
+  --color-white: rgba(var(--color-white--rgb));
```

Comment:
@naarok it looks like we are missing the transparency value here. Can we just use rgb instead?
```
  --color-white: rgb(var(--color-white--rgb));
```